src/components/Header/index.jsx

```diff
@@ -20,7 +22,7 @@ const Header = () => {
               <i className="fa fa-search"></i>
               <input
                 type="search"
-                placeholder="Search videos..."
+                placeholder="Search quiz..."
                 className="input"
                 defaultValue=""
                 onClick={(e) => {}}
```

Comment:
You can remove the onclick handler, if it's not used

Reply:
I kept it for searching function